### DISCUSSION TITLE
[Request] Invoice Ninja 5 | Type: LXC / No Docker

### DISCUSSION THREAD

**Discussion by EternalWebmedia** [tteck/Proxmox#2773] [Ideas]
##########
Git Repo:
https://github.com/invoiceninja/invoiceninja
Suggested installation instructions:
https://forum.invoiceninja.com/t/install-invoice-ninja-v5-5-on-ubuntu-22-04/13272
##########

Hey folks,

I'd like to see Invoice Ninja as a script for Proxmox LXC.
I really like the way to install, manage und update the services.

I would be very happy if someone finds the time to create Invoice Ninja as a script.

![mockup_5](https://github.com/tteck/Proxmox/assets/1334128/bbbaa8ff-f77e-457d-ae1b-b8c3d0e2b8ae)

**Comment by tteck** [comment]
You can utilize the TurnKey LXC Appliances script to create a [Invoice Ninja LXC](https://www.turnkeylinux.org/invoice-ninja)

**Reply by EternalWebmedia** [reply]
I tried it a few times and always got an error message.

I also find it very difficult to install updates.

**Reply by jproz** [reply]
I generally haven't had good experiences with Turnkey either.  For what it's worth @EternalWebmedia  - I just used @tteck debian LXC and then followed this https://techviewleo.com/how-to-install-invoice-ninja-on-debian-12-bookworm/
It's up and running. I haven't done squat with it yet - but it worked.

**Reply by jproz** [reply]
Just add - I ended up having to do a major update to IN as the thing above didn't download v5 correctly which I only noticed when I couldn't get gateways working. But once I did installation from git to update IN, gateways worked. Again - I'm just playing around with it but it seems to be working.
